include-build/roborazzi-gradle-plugin/src/main/java/io/github/takahirom/roborazzi/RoborazziPlugin.kt

```diff
@@ -301,20 +301,20 @@ abstract class RoborazziPlugin : Plugin<Project> {
         .configureEach { test ->
           val resultsDir = resultDirFileProperty.get().asFile
           if (restoreOutputDirRoborazziTaskProvider.isPresent) {
-            test.inputs.dir(restoreOutputDirRoborazziTaskProvider.map {
+            test.inputs.files(restoreOutputDirRoborazziTaskProvider.map {
```

Comment:
Could you please add a comment and a link here to explain why we use files instead of a directory?

Reply:
Done! Btw, I believe we can now clean up the following three lines that create the output dir if it doesn't exist -- let me do that in a follow up though.